types/googlepay/googlepay-tests.ts

```diff
@@ -92,6 +92,15 @@ function addGooglePayButton() {
     // $ExpectError
     buttonOptions.buttonSizeMode = 'unknown';
 
+    buttonOptions.buttonRootNode = undefined;
+    buttonOptions.buttonRootNode = document;
```

Comment:
What's the purpose of the re-assignments?

Reply:
This is just for tests. Making sure that expected values can be assigned and unexpected ones can't.